application/frontend/web/index.php

```diff
@@ -1,16 +1,5 @@
 <?php
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
 
-require(__DIR__ . '/../../vendor/autoload.php');
-require(__DIR__ . '/../../vendor/yiisoft/yii2/Yii.php');
-require(__DIR__ . '/../../common/config/bootstrap.php');
-require(__DIR__ . '/../config/bootstrap.php');
+$application = require('load-app.php');
```

Comment:
Yeah I think moving it back into index is probably better. Otherwise should add rule to .htaccess to prevent accessing load-app.php and maybe a check inside that file itself to make sure it is not being requested directly. Probably easier just to bring back into index.